brigade.js

```diff
@@ -102,26 +102,23 @@ function runSuite(e, p) {
   // throw it so the whole build will fail.
   //
   // Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#Promise.all_fail-fast_behaviour
-  //
-  // Note: as provided language string is used in job naming, it must consist
-  // of lowercase letters and hyphens only (per Brigade/K8s restrictions)
```

Comment:
Removed this comment because it's a remnant that became defunct after some previous round of refactoring.